text/0000-buildpack-versions.md

```diff
@@ -0,0 +1,62 @@
+# Meta
+[meta]: #meta
+- **Name:** Semantic Version Formats for Buildpack Versions
+- **Start Date:** 2020-03-11
+- **Supersedes:** N/A
+
+# Summary
+[summary]: #summary
+
+The Buildpacks team wants to enforce format requirements for buildpack versions but will not require version semantics. This RFC explains that buildpack authors are expected to use [semantic version](https://semver.org/) format without enforcing any semantic versioning rules.
+
+# Motivation
+[motivation]: #motivation
+
+There aren't specific guidelines in the buildpack documentation or spec for buildpack authors that explain how versioning works for buildpacks. It should be explicitly stated that the API expects a semantic version format and that authors are not required to follow semantic version semantics.
+
+Additionally, there are benefits of having a defined format because tools, such as `pack`, will be able to sort and filter buildpacks as needed. For example, with an expected semantic version format, a `latest` version is defined.
```

Comment:
@hone I added an additional part about being able to sort and filter buildpacks in the future.